tools/autosuspend-update.sh

```diff
@@ -4,4 +4,12 @@ set -eu
 cd "$1"
 
 (curl --fail -L 'https://chromium.googlesource.com/chromiumos/platform2/+/master/power_manager/udev/gen_autosuspend_rules.py?format=TEXT'; echo) \
-    | base64 -d > gen_autosuspend_rules.py
+    | base64 -d > tools/chromiumos/gen_autosuspend_rules.py
+
+(cat <<%EOF
+# This file is part of systemd.
+#
+# Pulled from libfprint upstream in order to autosuspend fingerprint readers
```

Comment:
should probably mention the license/copyright holder of the source of this in some way. better mention this too much then too little.

Reply:
Good point. We shuffled this around upstream and also added a SPDX license header there as that seemed more sensible than adding it on the systemd side.